gui/src/app/components/search-results/search-results.js

```diff
@@ -489,6 +493,7 @@ class SearchResults extends React.Component {
     let state = this.state;
     state.showCommunity = !this.state.showCommunity;
     this.setState(this._generateState(state));
+    this.analytics.addCategory('Toggle Community').sendEvent(this.props.analytics.VIEW);
```

Comment:
maybe add a label here to show if it's opening or closing?

Reply:
Oops, that's a bug, I meant to only send it when those are shown.